c++/src/kj/compat/http.c++

```diff
@@ -679,9 +679,16 @@ void HttpHeaders::addNoCheck(kj::StringPtr name, kj::StringPtr value) {
       indexedHeaders[id->id] = value;
     } else {
       // Duplicate HTTP headers are equivalent to the values being separated by a comma.
-      auto concat = kj::str(indexedHeaders[id->id], ", ", value);
-      indexedHeaders[id->id] = concat;
-      ownedStrings.add(concat.releaseArray());
+      if (strcasecmp(name.cStr(), "set-cookie") == 0) {
```

Comment:
Would a unit test for this branch make sense?

Reply:
ohhhh kay.